.github/workflows/docker-buildx.yml

```diff
@@ -0,0 +1,45 @@
+name: Build and publish docker image
+
+on:
+  workflow_dispatch:
+  workflow_run:
+    workflows: ["Release"]
+    types:
+      - completed
+
+jobs:
+  buildx_and_push_to_registry:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - uses: docker/setup-qemu-action@v1
+      - uses: docker/setup-buildx-action@v1
+
+      - name: Get the version
+        id: version
+        run: |
+          version=$(echo ${GITHUB_REF/refs\/tags\//} | cut -c 2-)
+          echo ::set-output name=version::$version
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Build and publish docker image
+        uses: docker/build-push-action@v2
+        with:
+          context: docker
+          file: ./docker/Dockerfile.release
+          build-args: RELEASE=${{ steps.version.outputs.version }}
+          platforms: linux/amd64,linux/arm64,linux/arm/v7
+          tags: |
+            docker.io/alephium/alephium:${{ steps.version.outputs.version }}
```

Comment:
Will this create 3 images with the same name ?

Reply:
Yes, exact. In docker hub, the tag will be shown once, but with multiple arch. See https://hub.docker.com/repository/docker/touilleio/alephium-mining-companion/tags?page=1&ordering=last_updated as an example

Reply:
thanks for the explanation. LGTM !